### DISCUSSION TITLE
[IntelliJ] New version 1.1.24.1610 breaks on Windows 7 due to NodeJS

### DISCUSSION THREAD

**Discussion by My2ndAngelic** [community/community#17800] [Copilot Conversations]
Hello,

This report is regarding the Github Copilot with IntelliJ. AFAIK, the last version of GitHub Copilot to work with Windows 7 is the version [1.1.22.1530](https://plugins.jetbrains.com/plugin/17718-github-copilot/versions/stable/175432).  However, the new version [1.1.24.1610](https://plugins.jetbrains.com/plugin/17718-github-copilot/versions/stable/180203) will break due to one API that is required in WS2_32.dll called "GetHostNameW", which is not present in this file on Windows 7.  I have tried launching copilot-agent-win.exe in the folder and that is exactly what it gave me.

Hopefully Windows 7 users can get some love from Copilot developers.

All the best.

**Comment by davecheney** [comment]
Windows 7 passed end of life two years ago https://www.microsoft.com/en-au/windows/windows-7-end-of-life-support-information#:~:text=After%2010%20years%2C%20support%20for,ended%20on%2014%20January%202020. It is unlikely it will be supported by the NodeJS developers so there isn't much we can do.
